packages/netlify-cms-core/src/components/Editor/Editor.js

```diff
@@ -210,8 +207,7 @@ export class Editor extends React.Component {
       const values = deserializeValues(entry.get('data'), fields);
       const deserializedEntry = entry.set('data', values);
       const fieldsMetaData = this.props.entryDraft && this.props.entryDraft.get('fieldsMetaData');
-      const mediaFiles = this.props.entryDraft && this.props.entryDraft.get('mediaFiles');
-      this.createDraft(deserializedEntry, fieldsMetaData, mediaFiles);
+      this.createDraft(deserializedEntry, fieldsMetaData);
```

Comment:
`mediaFiles` are part of `draftEntry.entry` so no need to pass them around separately